### PR TITLE
code reformat, readability, standards

### PR DESCRIPTION
**Pull request by markushausammann**
Changes are not well visible in the diff but it's just code formatting, whitespace, intendation, etc.

### CODE REVIEW

File: src/DocRaptor.php [hunk 1]
```diff
@@ -4,130 +4,154 @@
...
+                $fields['doc[prince_options][base_url]'] = $this->base_url;
+            }
+
+            if (!empty($this->document_content)) {
+                $fields['doc[document_content]'] = urlencode($this->document_content);
+            } else {
+                $fields['doc[document_url]'] = urlencode($this->document_url);
+            }
+
+            $fields_string = '';
```
Comment (janxious) [inline src/DocRaptor.php:123]: https://github.com/expectedbehavior/php-docraptor/pull/5 ?

**Comment by janxious** [comment]
:shipit:

File: src/DocRaptor.php (hunk 1 above)
Comment (markushausammann) [inline src/DocRaptor.php:123]: Seems you didn't merge this or?

Comment (janxious) [inline src/DocRaptor.php:123]: Just (poorly) asking if that's reasonable to incorporate.

Comment (markushausammann) [inline src/DocRaptor.php:123]: :) ah, yes, seems good

Comment (markushausammann) [inline src/DocRaptor.php:123]: it's perfect in fact :)

**Comment by janxious** [comment]
Merged #5 so feel free to merge this once you deal with the conflict.

**Pull request closed without merging**